src/app/online-tracker/online-tracker.service.spec.ts

```diff
@@ -0,0 +1,78 @@
+/* tslint:disable:no-unused-variable */
+
+import { TestBed, async } from '@angular/core/testing';
+import { Observable } from 'rxjs/Rx';
+import { Response } from '@angular/http';
+import { SessionService } from '../openmrs-api/session.service';
+
+import { OnlineTrackerComponent } from './online-tracker.component';
+import { OnlineTrackerService } from './online-tracker.service';
+
+describe('Service: OnlineTracker', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        OnlineTrackerService,
+        {
+          provide: OnlineTrackerService, useFactory: () => {
+            return new OnlineTrackerService(null);
+          }, deps: []
+        }
+      ]
+    });
+  });
+
+  afterEach(() => {
+    TestBed.resetTestingModule();
+  });
+
+  it('should create an instance', () => {
+    let service: OnlineTrackerService = TestBed.get(OnlineTrackerService);
+    expect(service).toBeTruthy();
+  });
+
+  it('should return true when online', () => {
+    let service: OnlineTrackerService = TestBed.get(OnlineTrackerService);
+    this.service.isOnline = true;
```

Comment:
@jknowles-cs  Why are calling this.service.isOnline? That should be service.isOnline.